runtime/doc/starting.txt

```diff
@@ -453,8 +453,9 @@ accordingly, proceeding as follows:
 	set or when using $VIMINIT.
 
      c. If the 'exrc' option is on (which is NOT the default), the current
-	directory is searched for two files.  The first that exists is used,
-	the others are ignored.
+	directory is searched for the following files.  The first that exists
```

Comment:
```suggestion
	directory is searched for the following files, in order of precedence.  The first that exists
```

Reply:
Mind if I change it a bit more?

```
     c. If the 'exrc' option is on (which is NOT the default), the current
	directory is searched for the following files, in order of precedence:
	- ".nvimrc.lua"
	- ".nvimrc"
	- ".exrc"
	The first that exists is used, the others are ignored.
```

Reply:
Let me know if I should change the file names in this PR. Or if it'll be decided and renamed later.

Personally, I don't mind the `rc` suffix. About other editors... `.idea`, `.vscode` are both directory. Also, if in future neovim wants to introduce a directory like that (e.g. `.nvim/`), renaming `.nvimrc` file to `.nvim` now would conflict with that later.

Reply:
>The "The file" before every line is kinda repetitive.

I agree, though I didn't suggest this because I wasn't sure how important it is to minimize the diff with the original Vim help files (to make porting patches easier). If that's not a concern, I second dropping "The file".

Reply:
> If that's not a concern

I can revert it if you want.

Btw, checked the vim's file. Looks like it has some other things going on, like separate naming for windows?

<img width="687" alt="image" src="https://user-images.githubusercontent.com/8050659/208152240-dbed5ef0-a9b9-4944-8a0f-249be10211a6.png">

So this part of the doc looks like already diverged a fair bit.

Reply:
No need to match vim here.